dictionaries/clojure/cspell-ext.json

```diff
@@ -0,0 +1,44 @@
+// cSpell Settings
+{
+    "id": "clojure",
+    "version": "0.2",
+    "name": "clojure",
+    "description": "Clojure dictionary for cspell.",
+    "readonly": true,
+    // List of dictionary files to add to the global list of dictionaries
+    "dictionaryDefinitions": [
+        {
+            "name": "clojure",
+            "path": "./clojure.txt.gz",
+            "description": "Clojure dictionary for cspell."
+        }
+    ],
+    // Dictionaries to always be used.
+    // Generally left empty
+    "dictionaries": [],
+    // Language Rules to apply to matching files.
+    // Files are matched on `languageId` and `local`
+    "languageSettings": [
+        {
+            // VSCode languageId. i.e. typescript, java, go, cpp, javascript, markdown, latex
+            // * will match against any file type.
+            "languageId": "clojure",
+            // Language locale. i.e. en-US, de-AT, or ru. * will match all locales.
+            // Multiple locales can be specified like: "en, en-US" to match both English and English US.
+            "locale": "en",
```

Comment:
`"locale": "en"` tells the spell checker to only use it if the locale in English.
Please make it `"*"`.